src/hotspot/share/prims/jvmtiEventController.cpp

```diff
@@ -572,9 +572,9 @@ JvmtiEventControllerPrivate::recompute_thread_enabled(JvmtiThreadState *state) {
     // mark if event is truly enabled on this thread in any environment
     state->thread_event_enable()->_event_enabled.set_bits(any_env_enabled);
 
-    // pointer to JavaThread can be NULL for unmouted virtual thread
     if (state->get_thread() != NULL) {
-      // update the JavaThread cached value for thread-specific should_post_on_exceptions value
+      // Update the JavaThread or mounted virtual thread cached value for
+      // thread-specific should_post_on_exceptions value.
       bool should_post_on_exceptions = (any_env_enabled & SHOULD_POST_ON_EXCEPTIONS_BITS) != 0;
```

Comment:
The updated comment does not look correct, so I want to clarify. We update the JavaThread for both normal and virtual thread. However, virtual thread can be unmounted. There is no JavaThread to update in this case.
Then the updated comment (in your style) should say something like this:
```
      // The JavaThread for carrier or mounted virtual thread case.
      // Update the cached value for thread-specific should_post_on_exceptions value.
```
But I feel that the original comments were okay. :)

Reply:
I've updated to use the revised comment you proposed.